### ISSUE TITLE
semicolon rule incorrectly applied inside of interface declarations

### ISSUE THREAD

**Issue by nonplus** [palantir/tslint#1613]
### Bug Report
- **TSLint version**: `3.15.1`
- **TypeScript version**: `2.0.3`
- **Running TSLint via**: grunt-tslint
#### TypeScript code being linted

``` ts
interface PollLinkOptions {
    account: { domain: string },
    poll: { uid: string },
    emailToken: string
};
```

with `tslint.json` configuration:

``` json
{
    "rules": {
        "semicolon": [
            true,
            "always"
        ]
}
```
#### Actual behavior

```
(semicolon) routes/links.ts[2, 30]: Missing semicolon
(semicolon) routes/links.ts[3, 24]: Missing semicolon
(semicolon) routes/links.ts[4, 20]: Missing semicolon
```
#### Expected behavior

Should not be treated as an error.

**Comment by jeffcressman** [comment]
@nonplus was having the same issue and stumbled across the tslint option `ignore-interfaces`, which would give you a configuration of

```json
{
    "rules":  {
        "semicolon": [
            true,
           "always",
           "ignore-interfaces"
        ]
    }
}
```

Picked it up from here https://palantir.github.io/tslint/rules/semicolon/

**Comment by JoshuaKGoldberg** [comment]
Awesome, looks like this is fixed! 🎉
